account_asset_management_xlsx/report/report_account_asset_xlsx.py

```diff
@@ -0,0 +1,458 @@
+                    cell_format = resources["fm_table_bold"]
+
+                # Write formula
+                if need_to_write_summary_formula:
+                    cell_format = resources["fm_table_number_bold"]
+                    start_cell = xl_rowcol_to_cell(start_row_pos, col_pos)
+                    stop_cell = xl_rowcol_to_cell(stop_row_pos, col_pos)
+                    cell_formula_value = cell_value.format(start_cell, stop_cell)
+                    sheet.write_formula(
+                        row_pos, col_pos, cell_formula_value, cell_format
+                    )
+
+                    # append cell to summary section info
+                    summary_column_info[column].append(
+                        xl_rowcol_to_cell(row_pos, col_pos)
+                    )
+                else:
+                    if column == "date" and not is_sub_summary_section:
+                        cell_format = resources["fm_table_date"]
+                        cell_value = format_date(self.env, cell_value)
+
+                    sheet.write(row_pos, col_pos, cell_value, cell_format)
+            row_pos += 1
+        resources.update(
+            {
+                "row_pos": row_pos + 1,
+                "summary_column_info": summary_column_info,
+            }
+        )
+        return resources
+
+    def generate_report_summary(self, resources):
+        sheet = resources["sheet"]
+        summary_column_info = resources["summary_column_info"]
+        row_pos = resources["row_pos"]
+        summary_data = {
+            "name": self.env._("Total"),
+            "value": "=SUM({})",
+            "value_residual": "=SUM({})",
+            "salvage_value": "=SUM({})",
+            "amo_ant": "=SUM({})",
+            "amo_de_lan": "=SUM({})",
+            "cum_amo": "=SUM({})",
+            "val_nette": "=SUM({})",
+        }
+        cell_format = resources["fm_table_header_dark_grey"]
+        table_infos = self._get_table_infos(resources)
+        for col_index, column in enumerate(table_infos, 0):
+            col_pos = col_index
+            cell_value = summary_data.get(column, "")
+            if "=SUM" in cell_value:
+                cell_format = resources["fm_table_number_bold_dark_grey"]
+                cell_formula_value = cell_value.format(
+                    ", ".join(summary_column_info[column])
+                )
+                sheet.write_formula(row_pos, col_pos, cell_formula_value, cell_format)
+            else:
+                sheet.write(row_pos, col_pos, cell_value, cell_format)
+        resources.update(
+            {
+                "row_pos": row_pos + 1,
+                "summary_column_info": summary_column_info,
+            }
+        )
+        return resources
+
+    def _get_created_by_info(self, wizard_rec):
+        today = fields.Date.today()
+        formated_today_str = format_date(self.env, today)
+        created_by_header = self.env._("Created by : ")
```

Comment:
```suggestion
        created_by_header = self.env._("Created on : ")
```
My mistake in my previous comment. It's `Created on`, not `Created by`